documentation/config.md

```diff
@@ -1,7 +1,30 @@
 
```

Comment:
As general rule, when a PR is implementing or fixing something not included in the previous relelease, we add one (or more) entries to CHANGES_NEXT_RELEASE. That file is used at release time to generate the changelog for the new release.

For instance, in this case something like that could be added:

```
- Add: environment variable based configuration (PERSEO_FE_URL and MAX AGE)
```

If the implemented/fixed item has an issue number associated then it is also included (maybe it doesn't applies this case, but take it into account in the future ;)

```
- Add: environement variable based configuration (PERSEO_FE_URL and MAX AGE) (#xxxx)
```

Reply:
Ok. I will remember
- Add: environment variable based configuration (PERSEO_FE_URL and MAX AGE)
:)